### DISCUSSION TITLE
Restreindre le mini-event d'augmentation des points d'affection aux familiers non dressés uniquement

### DISCUSSION THREAD

**Discussion by Nargacaura** [Crownicles/Crownicles#1322] [Idées acceptées / déjà implémentées]
On sait bien qu'un pet dressé ne perd plus en affection, du coup, tomber sur un mini-event de ce type revient à en avoir un n'ayant aucun effet... Du coup, pourquoi ne pas restreindre ce mini-event aux pets non dressés?

**Comment by BastLast** [comment]
Je valide

**Comment by BastLast** [comment]
Si qqun sur pc peut convertir en issue
